doc/source/cluster/kubernetes/examples/modin-example.md

```diff
@@ -0,0 +1,35 @@
+(kuberay-modin-example)=
+
+# Use Modin with Ray on Kubernetes
+
+In this example, you will learn how to run a modified version of the [Using Modin with the NYC Taxi Dataset](https://github.com/modin-project/modin/blob/4e7afa7ea59c7a160ed504f39652ff23b4d49be3/examples/jupyter/Modin_Taxi.ipynb) example from the Modin official repository using RayJob on Kubernetes.
+
+## Step 1: Install Kuberay operator
+
+Follow the steps 1 and 2 from the [RayCluster Quick Start](kuberay-raycluster-quickstart) guide to install Kuberay operator.
+
+## Step 2: Run the Modin example with RayJob
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-job.modin.yaml
+```
+
+## Step 3: Check the output
+
+Run the following command to check the output:
+
+```sh
```

Comment:
Have you ever updated the logs after unwrapping the Modin example from a Ray task?

Reply:
Done